packages/schema/bind/src/bindings/wasm-as/templates/imported/query-type/index-ts.mustache

```diff
@@ -2,6 +2,13 @@ import {
   w3_query,
   Nullable
 } from "@web3api/wasm-as";
+{{#methods}}
+{{#arguments}}
+{{#object}}
+import { {{type}} } from "../../{{type}}";
```

Comment:
Started working on schema-bind and have a question on how do we handle object imports in generated code?
I need to import all used objects in query method arguments and return from a single query
and make them unique per file (eg. CustomType is used in 2 methods) and separate them as user objects and imported objects as imports are not the same (imported objects are in separate folder).

What is the best way to handle this parsing and where is the most logical place to put it?

 - separate it after parseSchema in bind to required fields for importing to be language specific
 - use transfomer to alter query definition when parsing schema in bind to add required fields to query definition
 - handle it during parse and add requiredTypes to query definition though I don't think that is relevant to parsing stage


All ideas are welcome :smile: